### ISSUE TITLE
Troubleshoot Google Scholar indexing for the OAR

### ISSUE THREAD

**Issue by jrgriffiniii** [pulibrary/dspace-cli#80]
The measures for this were provided by a consultant with Atmire:

```
[...] if you block /jspui/browse path via robots.txt with Disallows (which I see you are) and/or remove linking to recent submission listings in your site, this can cause items to not get added to the index.
[...]
One alternative to unblocking browse might be to provide a link to the html sitemap somewhere in your homepage and see if they might start to crawl that as you listing. If that does not work ,then you may need to open up your browse pages to googlebot.
```

**Comment by jrgriffiniii** [comment]
Duplicate of #57
